guides/source/form_helpers.md

```diff
@@ -416,7 +418,7 @@ Notice that the appropriate option was automatically marked `selected="selected"
 
 ### Time Zone and Country Select
 
-To leverage time zone support in Rails, you have to ask your users what time zone they are in. Doing so would require generating select options from a list of pre-defined [`ActiveSupport::TimeZone`](https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html) objects using `collection_select`, but you can simply use the `time_zone_select` helper that already wraps this:
```

Comment:
At this point in the documentation `collection_select` has not yet been introduced.  Also, since `collection_select` is not "required" (as it describes) to generate the options, it seemed better to remove it.